.travis.yml

```diff
@@ -2,7 +2,7 @@
 before_install:
  - sudo add-apt-repository -y ppa:codegear/release
  - sudo apt-get -qq update
- - sudo apt-get install premake4
+ - sudo apt-get install premake4 uuid-dev
```

Comment:
What do you guys think of adding this dependency? Personally I'd rather avoid it unless we are reasonably confident that it will be available out of the box (without needing an extra install step) on a significant majority of systems.